nationalsCodev1.ino

```diff
@@ -1,85 +1,95 @@
-///Code Written by Nilai Vemula/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-//VERY IMPORTANT SETUP INFO FIRST///////////////////////////////////////////////////////////////////////////////
-  //Values to enter//
+/* Stuff to change at competition */
+// Distance where car should switch from fast to slow
+const long CHANGE_SPEED_DISTANCE = 8.5;
+// Final distance
+const long FINAL_DISTANCE = 9.5;
 
-const long change_speed_distance = 8.5;    ////Distance at where the car should switch from fast to slow (8.5m) ////
-const long final_distance = 9.5;     ////INSERT FINAL DISTANCE ////
-/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-
-//Wiring Information///////////////////////////////////////////////////////////////////////////////////////////////
-  //Encoder//
+/* Wiring Information */
 
+  /* Encoder */
     // Red - 5V
     // Black - GND
-    //Signal A in D2
-    const int encoder_a = 2; 
-    //Signal B in D3
-    const int encoder_b = 3; 
+    // Signal A in D2
+    const int ENCODER_A = 2; 
+    // Signal B in D3
+    const int ENCODER_B = 3; 
 
-  //Motor Shield//
+  /* Motor Shield */
 
-    // Motor PWM is connected to pin 5 (Check jumper).
-    const int pinPwm = 5;
-    // Motor DIR is connected to pin 4 (Check jumper).
-    const int pinDir = 4;
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    // Motor PWM - connected to pin 5 (Check jumper).
+    const int PIN_PWM = 5;
+    // Motor DIR - connected to pin 4 (Check jumper).
+    const int PIN_DIR = 4;
 
-//Setting initial values to zero//
-long encoder = 0;
-long REAL_DISTANCE = 0;
+// Set initial values
+void reset() {
+  long ENCODER_VALUE = 0;
+  long REAL_DISTANCE = 0; 
+}
 
-//Setting everything up//
+// Setup function
 void setup() {
-Serial.begin(9600); // Prepares the serial outputter //
-pinMode(encoder_a, INPUT_PULLUP); //Configures the encoder signal as an input//
-pinMode(encoder_b, INPUT_PULLUP); //Configures the encoder signal as an input//
 
-attachInterrupt(0, encoderPinChangeA, CHANGE); //Configures the encoder signal as an "interrupt"//
-attachInterrupt(1, encoderPinChangeB, CHANGE); //Configures the encoder signal as an "interrupt"//
+  // Call the reset
+  reset();
+
+  // Error handling
+  Serial.begin(9600);
 
-pinMode(pinPwm, OUTPUT); //Configures motor speed as output//
-pinMode(pinDir, OUTPUT); //Configures motor direction as output//
+  // Attach encoders
+  pinMode( ENCODER_A, INPUT_PULLUP );
```

Comment:
https://www.arduino.cc/en/Tutorial/InputPullupSerial
It uses the internal pull-up resistor of the arduino. This allows it to read a signal and then output to the serial monitor.